src/core/functions/aggregate/llm_min.cpp

```diff
@@ -0,0 +1,138 @@
+#include <inja/inja.hpp>
+#include <nlohmann/json.hpp>
+#include "flockmtl/core/module.hpp"
+#include "templates/llm_min_prompt_template.hpp"
+
+#include <flockmtl/common.hpp>
+#include <flockmtl/core/functions/aggregate.hpp>
+#include <flockmtl/core/functions/aggregate/llm_selector.hpp>
+#include <flockmtl/core/functions/prompt_builder.hpp>
+#include <flockmtl/core/model_manager/model_manager.hpp>
+#include <flockmtl_extension.hpp>
+
+namespace flockmtl {
+namespace core {
+
+struct LlmMinOperation {
```

Comment:
Let's merge this with the max by having an `LlmMinOrMaxOperation` with the difference between LlmMin and Llm max being the meta-prompt passed in the ctor of this base class.